playbooks/roles/notre-dame-nexus-dashboard/templates/ecosystem.yml.j2

```diff
@@ -0,0 +1,44 @@
+apps:
+  - name      : ndp
+    script    : {{ nexus_dashboard_code_dir }}/src/server.js
+    cwd       : {{ nexus_dashboard_code_dir }}
+    exec_mode : fork
+    env       :
+      NODE_ENV    : 'staging'
+      APP_URI     : 'http://{{ nexus_dashboard_host }}:{{ nexus_dashboard_port }}/'
+      MC_HOST     : '0.0.0.0'
+      MC_PORT     : '{{ nexus_dashboard_port }}'
+      DB_MAIN_URI : 'mongodb://{{ nexus_dashboard_mongo_host }}:{{ nexus_dashboard_mongo_port }}/ndplatform'
+      CH_MAIN_URL : 'redis://{{ nexus_dashboard_redis_host }}:{{ nexus_dashboard_redis_port }}'
+      WE_IDP_CS   : '{{ nexus_dashboard_idp_client_secret }}'
+      WE_IDP_RU   : 'http://{{ nexus_dashboard_host }}:{{ nexus_dashboard_port }}/auth/idp/callback'
+      WR_GOOGLE_CS: none
```

Comment:
I think we need a variable to set this, right? Similarly for many of the other variables here.

Another option to render this file is to create a dictionary in Ansible first, e.g.

    nexus_dashboard_configuration:
      apps:
        - name: ndp
          [...]
        - name: ndw
          [...]

For rendering the file, you can then use

    - name: Create ecosystem.yml file
      copy:
        content: "{{ nexus_dashboard_configuration | to_nice_yaml }}"
        dest: /notredame/apps/ecosystem/ecosystem.yml

This would avoid all the boilerplate for the configuration variables – we would basically write the whole configuration file in the Ansible variables instead.